.github/workflows/deploy-workflow.yaml

```diff
@@ -13,11 +13,11 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: '18'
+          node-version: '20'
       - run: npm install
       - run: npm run build
         env:
-          REACT_APP_BITLY_KEY: ${{ secrets.REACT_APP_BITLY_KEY }}
+          VITE_BITLY_KEY: ${{ secrets.VITE_BITLY_KEY }}
```

Comment:
I think this means we'll need to remember to update the SECRETS in the GitHub settings too, right?

Reply:
Yep. I already added a new secret to Github. I can remove the old one once this is deployed since it won't be needed anymore.